helm/ffc-demo-payment-service-core/values.yaml

```diff
@@ -46,9 +46,11 @@ container:
   comPlusEnableDiagnostics: 0
 
 postgresService:
-  postgresConnectionString: User Id=postgres;Server=ffc-demo-payment-core-postgres;Port=5432;Database=ffc_demo_payments;Password=ppp;
+  postgresConnectionStringAlt: dummy
+  postgresConnectionString: dummy
```

Comment:
Is there a need to split these or is this left over from testing?

Reply:
Connection string has been replaced by individual username, password, host etc settings for the DB